gk/main.c

```diff
@@ -917,20 +937,25 @@ process_pkts_front(uint8_t port_front, uint8_t port_back,
 
 		if (ret < 0)
 			rte_pktmbuf_free(pkt);
```

Comment:
It would be worth replacing all calls to `rte_pktmbuf_free()` with `drop_packet()` in the GK blocks. This should help us when we start to collect statistics.